src/platform/nm-platform.c

```diff
@@ -6369,17 +6369,40 @@ const char *
 nm_platform_qdisc_to_string (const NMPlatformQdisc *qdisc, char *buf, gsize len)
 {
 	char str_dev[TO_STRING_DEV_BUF_SIZE];
+	char opt_buf[300] = { 0, };
+	char *p = opt_buf;
+	gsize l = sizeof (opt_buf);
 
 	if (!nm_utils_to_string_buffer_init_null (qdisc, &buf, &len))
 		return buf;
 
-	g_snprintf (buf, len, "%s%s family %d handle %x parent %x info %x",
+	if (nm_streq (qdisc->kind, "fq_codel")) {
```

Comment:
we must also to-string ~invalid~ (or at least ~incomplete~) instances. `nm_streq0()`.